docker/docker-compose-redis.yml

```diff
@@ -0,0 +1,11 @@
+#
+# This file is a placeholder for a docker compose file that will start the rh-ui.
```

Comment:
This should be mentioned in the README for part of the setup for running locally.

Reply:
This is implemented and documented properly in Peter's changes - I'll do a proper review of the Readme when both changes are merged

Reply:
ok sounds fine